CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# magma-orchestrator-interface
+
+## Developing
+
+Create and activate a virtualenv with the development requirements:
+
+    virtualenv -p python3 venv
+    source venv/bin/activate
+
```

Comment:
If I'm not mistaken tox will create a virtual env.
```suggestion
You can use the environments created by `tox` for development:

```bash
tox --notest -e unit
source .tox/unit/bin/activate
```
```

Reply:
Okay :)